### PR TITLE
Converting C# Basics to use .NET Fiddle

### PR DESCRIPTION
**Pull request by webreidi**

### CODE REVIEW

File: content/english/csharp-basics/comments.md [hunk 1]
```diff
@@ -18,6 +18,13 @@ Also, adding comments will not affect the program whatsoever. So, you can add co
       * multi-line
       * comment */
 ```
+
+## Play with Comments
+
+<a class="my-2 mx-4 btn btn-info" href="https://replit.com/@nuevofoundation/CSharpBasicsHelloWorld" target="_blank">Launch Replit</a>
```
Comment (sunnynagavo) [inline content/english/csharp-basics/comments.md:24]: Is this still replit ?

File: content/english/csharp-basics/variables.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ boolean loveMusic = true;
 
 The last varable is an interesting one.  It can only have a value of `true` or `false`.  We will see that this can be very helpful in the more advanced sections of this workshop.
 
-# Play with variables
+## Play with variables
 
 Let's see what we can do with variables. Launch and fork the repl.
```
Comment (sunnynagavo) [inline content/english/csharp-basics/variables.md:58]: one more for replit ?

File: content/english/csharp-basics/variables.md [hunk 3]
```diff
@@ -52,7 +52,7 @@ boolean loveMusic = true;
...
```
Comment (sunnynagavo) [inline content/english/csharp-basics/variables.md:57]: Typo: fork the repo ?

File: content/english/csharp-basics/variables.md [hunk 4]
```diff
@@ -64,7 +64,7 @@ Try entering these lines and **run**:
 int age = 10;
 Console.WriteLine(age); // prints out 10
 age = 12; // assign data 12 to the variable age
-Console.WriteLine(age); // prints out 10
+Console.WriteLine(age); // prints out 12
 ```
 We can assign a value to a varable, reference it and then change the value.
```
Comment (sunnynagavo) [inline content/english/csharp-basics/variables.md:70]: One more replit link at the bottom

File: content/english/csharp-basics/using-netfiddle.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+<a class="my-2 mx-4 btn btn-info" href="https://dotnetfiddle.net/SignUp" target="_blank">Sign up for .NET Fiddle</a>
+<img src="../images/dotnetfiddle-signup.png" height="300" alt=".NET Fiddle sign up form" />
+
+Next, launch the .NET Fiddle console we will use today by clicking the below button. You should see the screen below.
+
+<a class="my-2 mx-4 btn btn-info" href="https://dotnetfiddle.net/oLKxyN" target="_blank">Launch .NET FIddle</a>
+
+<img src="../images/dotnetfiddle-launchScreen.png" height="300" alt="The starting screen of a .NET Fiddle" />
+
+We will now *fork* the Repl, which simply means to make a copy of the program, so that we can make our own changes to it. Click on the "Fork" button on the top.
```
Comment (veronicagg) [inline content/english/csharp-basics/using-netfiddle.md:23]: fork the Repl-> Fiddle ?

File: content/english/csharp-basics/using-netfiddle.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+After forking the Replit, you will see a window similar to the one below:
```
Comment (veronicagg) [inline content/english/csharp-basics/using-netfiddle.md:25]: Replit -> Fiddle

File: content/english/csharp-basics/classes-and-objects.md [hunk 7]
```diff
@@ -42,10 +42,6 @@ Let's learn about the different parts in a class below:
 ### Class
 A class is a blueprint or prototype of a new type of object. In general, a class contains three important parts: 
 
-* data fields/instance variables 
-* constructor 
-* methods
-
 <img src="../images/class.png" height="400" alt="A Class blueprint for an object contains instance varialbes/data fields which are data/attributes in the object, constructor which are methods that creates the obejct of the class, and methods, which are behaviors possible for the object."/>
```
Comment (veronicagg) [inline content/english/csharp-basics/classes-and-objects.md:45]: does the image show up? I believe there may be a extra "." in the path which causes a broken image

File: content/english/csharp-basics/classes-and-objects.md [hunk 8]
```diff
@@ -81,17 +94,17 @@ Let's make a `Bird` class to represent Patrick 🐥 and all of his Bird friends
...
 ```
 
 <br />
 
 ### Second, let's declare all the fields of the Bird class: species, name, hobby, age, loveMusic.
 
-Each field is declared in the format: `private` `data type` `name` `;`.
+Each field is declared in the format: `access specifier` `data type` `name` `;`.
 
-1. We declare all 5 fields as `private` in the class `Bird`. This ensures these data can only be accessed within this class.
```
Comment (veronicagg) [inline content/english/csharp-basics/classes-and-objects.md:94]: replit reference above?

File: content/english/csharp-basics/classes-and-objects.md [hunk 9]
```diff
@@ -109,15 +122,15 @@ For example, to declare `species` as a private field of class `Bird`, you would
 
 Usually, constructor is the method that initialializes value to all the fields in a class. It has the format `public` `class name` `(parameter)`. Since we have 5 fields in this class, the constructor will take in 5 parameter/inputs.
```
Comment (veronicagg) [inline content/english/csharp-basics/classes-and-objects.md:123]: typo: initialializes -> initializes ?

File: content/english/csharp-basics/classes-and-objects.md [hunk 10]
```diff
@@ -207,8 +220,8 @@ Try it out yourself and create multiple Birds of different species 🐦🐤🐔
...
 
 1. Print the return value of `ToString()` method. (i.e. `Console.WriteLine(patrick.ToString());`)
-2. Directly print the `Bird` object, which prompt `ToString()` to be called in the background. (i.e. `Console.WriteLine(patrick);`).
+2. Directly print the `Bird` object, which prompts `ToString()` to be called in the background. (i.e. `Console.WriteLine(patrick);`).
 
 Try it out and print out all the information of the bird friends you created 🐦🐤🐔🐧!
 
-<a class="my-2 mx-4 btn btn-info" href="https://replit.com/@nuevofoundation/CSharpBasicsClasses" target="_blank">Launch Replit</a>
\ No newline at end of file
+<a class="my-2 mx-4 btn btn-info" href="https://replit.com/@nuevofoundation/CSharpBasicsClasses" target="_blank">Launch Replit</a>
```
Comment (veronicagg) [inline content/english/csharp-basics/classes-and-objects.md:227]: replit?

File: content/english/csharp-basics/control-structures.md [hunk 11]
```diff
@@ -65,7 +65,7 @@ An **if-else statement** follows the structure shown above. If the boolean expre
 The code above will print the statement `I like Music` because the boolean expression, `likeMusic == true`, is true.
 {{% /notice %}}
 
-Let's revisit the greating example above, and make decision for Patrick on whether to say "Good Morning," "Good Afternoon," "Good Night". 
+Let's revisit the greeting example above, and make decision for Patrick on whether to say "Good Morning," "Good Afternoon," "Good Night". 
 
 Patrick should say "Good Morning" between 0:00 - 11:59, "Good Afternoon" between 12:00 - 19:59, and "Good Night" between 20:00 - 23:59.
```
Comment (veronicagg) [inline content/english/csharp-basics/control-structures.md:71]: replit reference 5 lines down

File: content/english/csharp-basics/using-netfiddle.md [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+date: 2024-09-17T00:00:00Z
+weight: 1
+draft: false
+---
+
+## Launch .NET Fiddle
+
+Today you will be using .NET Fiddle to learn how to code in C#. 
+
+First, launch the .NET Fiddle console we will use today by clicking the below button. You should see the screen below.
```
Comment (ozhang22) [inline content/english/csharp-basics/using-netfiddle.md:12]: No longer relevant - everything will be embedded

**Review by ozhang22 — APPROVED** [review]

**Pull request merged**